collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -995,7 +1014,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
      * @return false if any error occurs while saving (constraint violated,
      * etc...), true otherwise.
      */
-    private boolean saveAnswersForCurrentScreen(boolean evaluateConstraints) {
+    private boolean evaluateConstraintsAndSaveAnswersForCurrentScreen() {
```

Comment:
Do we still need this method? We have `updateAnswersForScreen` in `FormEntryViewModel`.